subcommands/waves/list.go

```diff
@@ -44,4 +49,25 @@ func doListWaves(cmd *cobra.Command, args []string) {
 		)
 	}
 	t.Print()
+	if lst.Next != nil {
+		fmt.Print("\nNext page of devices can be viewed with: ")
+		found := false
+		for i := 0; i < len(os.Args); i++ {
+			arg := os.Args[i]
+			if (len(arg) > 2 && arg[:2] == "-p") || (len(arg) > 7 && arg[:7] == "--page=") {
+				fmt.Printf("-p%d ", showPage+1)
+				found = true
+			} else if arg == "-p" || arg == "--page" {
+				fmt.Printf("-p%d ", showPage+1)
+				found = true
+				i++
+			} else {
+				fmt.Print(os.Args[i], " ")
+			}
+		}
+		if !found {
+			fmt.Print("-p", showPage+1)
+		}
+		fmt.Println()
```

Comment:
I've already seen an almost identical code somewhere. It should be feasible to extract the common part into a function in e.g. `subcommands/common.go` and reuse in both places.

Reply:
Thanks